src/System.Management.Automation/namespaces/LocationGlobber.cs

```diff
@@ -2080,6 +2083,12 @@ internal bool IsShellVirtualDrive(string driveName, out SessionStateScope scope)
                         // this is the default behavior for all windows drives, and all non-filesystem
                         // drives on non-windows
                         path = path.Substring(driveName.Length + 1);
+
+                        if (path == "")
+                        {
+                            // path is to the root of a drive such as "C:\" or "C:"
```

Comment:
The path has to be like `c:` for this to happen not "c:\". Please update comment.